### ISSUE TITLE
Can't install packages via Lite Terminal

### ISSUE THREAD

**Issue by isc-tleavitt** [intersystems/ipm#579]
This is an upstream issue (ref: DP-435099), but I think we can provide a workaround as it'll be a while before this makes it into IRIS kits.

For example, on an environment with just git-source-control installed:

```
USER>zpm "list"
git-source-control 2.5.0
USER>w $stack
3
USER>d $System.SQL.Purge()
 
USER>w $stack
3
USER>zpm "list"
 
USER>w $stack
40
```

Issue happens in SQL compilation of:
`SELECT Nsp FROM %SYS.Namespace_List() WHERE Nsp = ?`

Workaround would be to implement the WHERE clause in ObjectScript outside of this query.

Should be fixed in PRs targeting both master and v1. We'll want a new 0.7 release.

**Comment by isc-tleavitt** [comment]
@isc-shuliu we need an equivalent targeting master as well

**Comment by isc-shuliu** [comment]
Master is fixed in #592
